### DISCUSSION TITLE
一个不知道什么项目的参考图

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#398] [General]
备份一下，说不定未来可以用上

![EDA12CD5-D5B4-4317-9160-D2790412142E](https://user-images.githubusercontent.com/13673702/141947378-9d1be236-1301-45df-b51f-a41353bd2415.jpeg)

**Comment by mark9804** [comment]
我后来重新设计了一个card页面，文字配色基本上也是金色，不过没想到渐变色

图很好，现在是我的了.jpg

**Comment by Arondight** [comment]
这个头像用的是 QQ 头像，有点意思

**Reply by mark9804** [reply]
QQ头像 ~~Silverlight~~ SilveryStar 那边实现了

**Reply by Arondight** [reply]
> QQ头像Silverlight那边实现了

这周末扒过来

**Reply by mark9804** [reply]
> 这周末扒过来

对了，HTML 里面的图像`src`[可以直接传 base64](https://stackoverflow.com/a/48216882)，这样我们实现的时候就不用考虑头像的路径问题了

如果有布局方面的问题解决不了就踢我一脚，我来弄，只要图能显示在页面上就可以

**Comment by Arondight** [comment]
一个角色详情参考图 https://github.com/Le-niao/Yunzai-Bot

**Comment by mark9804** [comment]
如果到时候有时间的话，我打算参考第一张图和这个来做：[玩游戏聊设计-《杀手3》极致奢华的黑金色迪拜](https://www.bilibili.com/video/BV1vb4y1b7i7)

我想尽可能不用 png 和 jpg 这类的点阵图片，全部用矢量图和 CSS 来完成想法，这样对于存储空间来说毫无压力，有压力的主要是头发

我打算等二月份学校放春假了做，这段缓冲时间正好看看米忽悠会不会又折腾API

**Comment by Arondight** [comment]
看到一个电报 bot 配色挺好

![D69ED658-1CFA-4975-A4A5-86B43EE63AE4](https://user-images.githubusercontent.com/13673702/143821463-ad9cf096-9dac-47b3-acec-a4e052cdddb2.jpeg)

**Comment by mark9804** [comment]
对对对我想要的就是这种效果（甲方拍大腿）
![2BA904B9-B54F-45A6-93AB-09C9EF569CCB](https://user-images.githubusercontent.com/9006264/148636738-96bb5b4b-2cc7-4ad0-8763-ad77751647e7.jpeg)
![image](https://user-images.githubusercontent.com/9006264/148636761-36c88152-c4fe-4690-bed5-130b2624ab59.png)
![image](https://user-images.githubusercontent.com/9006264/148636777-5982f225-212d-4c09-810c-4d138b77464f.png)

**Comment by mark9804** [comment]
找到一个疑似米忽悠 UI 设计师的主页，里面的废弃设计太多了，很难让人不觉得不是真舅舅

https://www.artstation.com/risingview

**Reply by mark9804** [reply]
多方求证，真是设计师本人
<img src="https://user-images.githubusercontent.com/9006264/152099720-8160cb71-857e-4d20-9b62-727c8738b159.png" width="200"/>

**Comment by mark9804** [comment]
https://fffuel.co/uuunion/

这个网站配色选项都不错，偷了

**Comment by mark9804** [comment]
两种金色

![image](https://user-images.githubusercontent.com/9006264/158515640-a8805513-6204-4ccd-ac45-7248cf0d82b2.png)
![image](https://user-images.githubusercontent.com/9006264/158517752-dc13bfbc-c104-48c4-843b-013cd667549f.png)
